extensions/blocks/opentable/opentable.php

```diff
@@ -91,7 +91,15 @@ function add_language_setting() {
 function load_assets( $attributes ) {
 	\Jetpack_Gutenberg::load_assets_as_required( FEATURE_NAME );
 
-	$classes = \Jetpack_Gutenberg::block_classes( FEATURE_NAME, $attributes );
+	$classes = array( sprintf( 'wp-block-jetpack-%s-theme-%s', FEATURE_NAME, get_attribute( $attributes, 'style' ) ) );
```

Comment:
Should we escape `get_attribute( $attributes, 'style' )`?

```suggestion
	$classes = array(
		sprintf(
			'wp-block-jetpack-%s-theme-%s',
			FEATURE_NAME,
			esc_attr( get_attribute( $attributes, 'style' ) )
		),
	);
```

Reply:
Good point! For some reason I thought we were already escaping things in `get_attribute`